security/network_bound_disk_encryption/nbde-disaster-recovery-considerations.adoc

```diff
@@ -0,0 +1,27 @@
+// CNF-2127 assembly
+[id="nbde-disaster-recovery-considerations"]
+= Disaster recovery considerations
+include::modules/common-attributes.adoc[]
+:context: nbde-implementation
+
+toc::[]
+
+This section describes several potential disaster situations and the procedures to respond to each of them. Additional situations will be added here as they are discovered or presumed likely to be possible.
+
+// include::modules/nbde-disaster-recovery-considerations.adoc[leveloffset=+1]
+
+include::modules/nbde-loss-of-a-client-machine.adoc[leveloffset=+2]
+
+include::modules/nbde-loss-of-client-connectivity.adoc[leveloffset=+2]
+
+include::modules/nbde-unexpected-loss-of-network-connectivity.adoc[leveloffset=+3]
```

Comment:
Unless you feel strongly, the `+3` on lines 17 to 21 could be `+2` and that'll make them part of the TOC at the top of the page.

Reply:
Tat would be better. done.